### DISCUSSION TITLE
Add a dropdown button to see a list of all open scenes

### DISCUSSION THREAD

**Discussion by soccerob** [godotengine/godot-proposals#7953] [Editor]
Rather than just using the left/right arrows to browse between open scenes in the 2D editor, I think it would be more useful to have a small dropdown icon next to the arrows. Upon clicking the dropdown, a list of all currently opened scenes are listed, clicking on an item in the list will open up that scene for editing. 

Using the arrows is fine when you only have a few scenes open, but when jumping into editing child scene instances, the tabs quickly outgrow the space allocated and locating a specific scene becomes overly tedious. I believe this would speed up searching for scenes and be an incredibly slim adjustment to the editor. 
![Godot Dropdown](https://github.com/godotengine/godot-proposals/assets/151275/69c5df8f-9b57-4a2d-9003-8c0297327501)

**Comment by LeHaHaDev** [comment]
This would be amazing! Similar to how VS handles single tab line file selection.
